### DISCUSSION TITLE
Welcome to Athena Discussions!

### DISCUSSION THREAD

**Discussion by mmills6060** [Banbury-inc/Athena#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by JY725** [comment]
Hello, @mmills6060. I'm a Senior Fullstack developer with 5+ years of experience in Web&App.
I have a rich knowledge of Python and Typescript.
I can help you.
